.github/workflows/apk.yaml

```diff
@@ -22,23 +20,23 @@ jobs:
         path: isaac/
 
 
-    - name: Build image isaac/astrobee:msgs-ubuntu16.04
+    - name: Build messages dockers for Ubuntu 20 (astrobee)
       run: docker build astrobee -f isaac/scripts/docker/astrobee_msgs.Dockerfile
-                                 --build-arg UBUNTU_VERSION=16.04
-                                 --build-arg ROS_VERSION=kinetic
-                                 --build-arg PYTHON=''
-                                 -t isaac/isaac:astrobee-msgs-ubuntu16.04
+                          --build-arg UBUNTU_VERSION=20.04
+                          --build-arg ROS_VERSION=noetic
+                          --build-arg PYTHON=3
+                          -t ghcr.io/${{ github.repository_owner }}/isaac:astrobee-msgs-ubuntu20.04
 
-    - name: Build image isaac/isaac:msgs-ubuntu16.04
+    - name: Build messages dockers for Ubuntu 20 (isaac)
       run: docker build isaac -f isaac/scripts/docker/isaac_msgs.Dockerfile
-                              --build-arg UBUNTU_VERSION=16.04
-                              --build-arg ROS_VERSION=kinetic
-                              --build-arg PYTHON=''
-                              -t isaac/isaac:msgs-ubuntu16.04
+                          --build-arg UBUNTU_VERSION=20.04
+                          --build-arg ROS_VERSION=noetic
+                          --build-arg PYTHON=3
+                          -t ghcr.io/${{ github.repository_owner }}/isaac:msgs-ubuntu20.04
```

Comment:
this is why the test is failing, it's expecting the previous image to have remote isaac/ instead of nasa/. You can add the argument:
`--build-arg REMOTE=ghcr.io/${{ github.repository_owner }}`